react-hot-button/my-app/src/App.tsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from 'react';
+import { HotButton } from './HotButton';
+import './App.css';
+
+export function App() {
+  const [btnClicks, setBtnClicks] = useState(0);
+  const [btnLvlClass, setBtnLvlClass] = useState('hot-btn-lvl-1');
```

Comment:
You should compute this from `btnClicks` rather than having it as separate state. That way you won't have to set both states in response to a click.